templates/catalog/product.html

```diff
@@ -41,81 +41,86 @@
-                    <div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-                      {{ product.price|format_price }}
-                      <meta itemprop="price" content="{{ product.price|floatformat:2 }}">
-                      <meta itemprop="priceCurrency" content="RUB">
-                    </div>
-                  </td>
-                  <td class="product-order-table-col-count">
-                    {% include 'catalog/count_input.html' %}
-                  </td>
-                  <td class="product-order-table-col-total">
-                    <span class="js-product-sum">{{ product.price|format_price }}</span>
-                  </td>
-                </tr>
-              </table>
-
-              <div class="public-offer product-order-public-offer">Не является публичной офертой</div>
-
-              <div class="product-order-submit">
-                <button class="btn buy-product" id="buy-product"
-                 data-name="{{ product.name }}"
-                 data-category="{{ product.category.name }}"
-                 data-id="{{ product.id }}"
-                 type="submit">Заказать</button>
-              </div>
-            </div>
-          </div>
+        <div class="option-order row">
+          <table class="options-table table">
+            <tr>
+              <th>Длина (мм)</th>
+              <th>Ширина (мм)</th>
+              <th>Высота (мм)</th>
+              <th>Внутренний диаметр (мм)</th>
+              <th>Масса (кг)</th>
+              <th>Объем (м<sup>3</sup>)</th>
+
+              <th>Цена</th>
+              <th>Кол-во</th>
+              <th>Итог</th>
+              <th></th>
+            </tr>
+            {% for option in product.options.all %}
+              <tr>
+                <td class="option-td">{{ option.length }}</td>
+                <td class="option-td">{{ option.width }}</td>
+                <td class="option-td">{{ option.height }}</td>
+                <td class="option-td">{{ option.diameter_in }}</td>
+                <td class="option-td">{{ option.weight|floatformat:'0' }}</td>
+                <td class="option-td">{{ option.volume }}</td>
+
+                <td class="option-td option-td-price js-option-price">
+                  <div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
+                    {{ option.price|format_price }}
+                    <meta itemprop="price" content="{{ option.price|floatformat:2 }}">
+                    <meta itemprop="priceCurrency" content="RUB">
+                  </div>
+                </td>
+
+                <td class="option-td">
+                  {% include 'catalog/count_input.html' %}
+                </td>
+                <td class="option-td">
+                  <span class="js-option-sum">{{ option.price|format_price }}</span>
+                </td>
+                <td class="option-td">
+                  {# @todo #335:30m  Adapt order button to Options. #}
```

Comment:
What exactly should we do with the button? Or is it part of the issue to find out this?

Reply:
i'll provide more description